packages/core/src/queries/log.ts

```diff
@@ -44,3 +44,17 @@ export const findLogs = async (limit: number, offset: number, logCondition: LogC
     limit ${limit}
     offset ${offset}
   `);
+
+export const getDnuCountsByTimeInterval = async (
```

Comment:
just wondering what's DNU stands for?

Reply:
Daily New User.

Added the comment for clarity. PTAL.

Reply:
dnu is ambiguous. would prefer full name.

Reply:
also i don't think we have the need of daily new user per design. just need two sets of numbers with a specific `date`:

1. new user count for `date` and `date.yesterday()`
2. new user count for `[date - 6 days, date]` and `[date - 13 days, date - 7 days]`

<img width="817" alt="image" src="https://user-images.githubusercontent.com/14722250/168804571-86f4be93-3d40-4130-94b5-453c4c8a85c9.png">

@simeng-li would you like to help and figure out the correct API design?